public/views/partials/organisms/content-card.liquid

```diff
@@ -62,15 +62,20 @@ metadata:
 
 {% capture card_content %}
   <div class="h-full flex flex-col">
-    <div>
+    <div class="relative w-full h-0 pt-[calc(100%*(9/16))]">
       {% if url != blank %}
       <a href="{{url}}">
       {% endif %}
         <picture>
-          {% if avifSrcSet != blank %}
-          <source type="image/avif" srcset="{{avifSrcSet}}">
+          {% if avifSrcSet != blank and imgSrc != blank %}
+            <source type="image/avif" srcset="{{avifSrcSet}}">
+            {% assign src = imgSrc %}
+            {% assign srcset = imgSrcSet %}
+          {% else %}
+            {% assign src = 'images/PCBS_-_Placeholder_image.png' | asset_url %}
```

Comment:
this creates a dependency between the components module and the pcbs app. this means that the module will only work for PCBS.
it should be the other way around: it's okay for an app to depend on a module, but it's never okay for a module to depend on the app.

Reply:
what would be the best way to solve this? having a constant as a configuration with the name of the placeholder? creating an `atom` for placeholder?